docker/dockerbuild/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM circleci/python:2.7.15-stretch
```

Comment:
why not `slim` version of stretch here too?

Reply:
This is from circleci. It is an image to use for building our images (dev only). It is not an image we will be shipping. Circle don't have a slim version from what I saw.